benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm

```diff
@@ -182,4 +182,4 @@ end
 
 subsection Checkpointing
   set Steps between checkpoint = 10
-end
+end
```

Comment:
Please readd the final newline character in this file. This was likely deleted by your VS code, see [here](https://stackoverflow.com/a/44704969) for how to prevent this in the future.